examples/alias/main.tf

```diff
@@ -25,17 +25,15 @@ module "lambda_function" {
 
   function_name = "${random_pet.this.id}-lambda"
   handler       = "index.lambda_handler"
-  runtime       = "python3.8"
+  runtime       = "nodejs14.x"
```

Comment:
Why do you change `runtime` and `source_path`? Is it required for the PR to work? I don't think so.

Reply:
The `python3.8` example doesn't work in case a developer use `pyenv` as the screenshot in the description so I changed to `nodejs14.x` example instead so when new developer contributes he/she won't face the same issue. I can revert back if you prefer.

Reply:
Reverted.